woocommerce/woocommerce
# Changed settings warning when leaving page

### All Submissions:






### Changes proposed in this Pull Request:



When there are changes in settings fields message "The changes you made will be lost if you navigate away from this page." should be shown always, not only when .woo-nav-tab-wrapper are clicked.

Closes # .

### How to test the changes in this Pull Request:

1. Go to woocommerce general setting
2. Click on any WordPress menu item
3. Alert "The changes you made will be lost if you navigate away from this page." should be shown

### Other information:




### Changelog entry

> Fix - Display warning hen leaving settings page without saving first.
